RadixWallet/Core/DesignSystem/Extensions/View+Extra.swift

```diff
@@ -9,21 +9,46 @@ extension View {
 			}
 	}
 
-	func setUpNavigationBar(title: String) -> some View {
+	func setUpNavigationBar(title: String, setBackgroundColor: Bool = true) -> some View {
 		self
 			.toolbar {
 				ToolbarItem(placement: .principal) {
 					Text(title)
 						.foregroundColor(.app.gray1)
-						.textStyle(.secondaryHeader)
+						.textStyle(.body1Header)
 				}
 			}
 			.navigationBarTitleDisplayMode(.inline)
-			.toolbarBackground(.app.background, for: .navigationBar)
-			.toolbarBackground(.visible, for: .navigationBar)
+			.applyIf(setBackgroundColor) {
```

Comment:
Can't we just use the Visibility parameter for `toolbarBackground` to enable/disable it, rather than `applyIf`?